rotkehlchen/api/rest.py

```diff
@@ -321,7 +323,10 @@ def _query_async(self, command: Callable, **kwargs: Any) -> Response:
         )
         greenlet.task_id = task_id
         greenlet.link_exception(self._handle_killed_greenlets)
-        self.rotkehlchen.api_task_greenlets.append(greenlet)
+        self.rotkehlchen.api_task_greenlets.append(ApiTaskGreenlet(
```

Comment:
General question. Isn't it missing the tracking and killing of periodic tasks?

Reply:
yes, adding that.